test/libsolidity/util/ContractABIUtils.cpp

```diff
@@ -200,6 +202,18 @@ bool ContractABIUtils::appendTypesFromName(
 )
 {
 	string type = _functionOutput["type"].asString();
+	FixedPointType const* fixedPointType = nullptr;
```

Comment:
Please move this into a function
````
std::optional<FixedPointType> isFixedPoint(string const& _type)
```
and combine the call with `https://github.com/ethereum/solidity/pull/11682/files#diff-cb738e27be4a73d349225f0912bb18590560ba7b704a32f6ca3c72d55bd6fdb5R262` (`else if (fixedPointType != nullptr)`)

Reply:
Actually why go the detour via `FixedPointType`? The function could also just return `std::optinoal<ABIType>`